spec/requests/providers/has_other_proceedings_controller_spec.rb

```diff
@@ -102,8 +102,8 @@
 
       let(:params) { { legal_aid_application: { has_other_proceeding: "false" } } }
 
-      it "redirects to the LASPO page" do
-        expect(response).to redirect_to(providers_legal_aid_application_in_scope_of_laspo_path(legal_aid_application))
+      it "no longer redirects to the LASPO page, but continues to delegated functions" do
```

Comment:
Should we care what the test used to test when we're reading it?

I'd be tempted to say

```ruby
it "redirects to the delegated functions page" do
```

and rely on the commit messages to tell the story of what changed and why.

What do you think?

Reply:
I'm all about the history! 😆 
Fair point - will get updated when I remove the temp redirect commit 👍